.github/workflows/charts.yml

```diff
@@ -0,0 +1,64 @@
+name: Charts
+
+on:
+  pull_request:
+    branches: [ main, release/** ]
+  push:
+    branches: [ main, release/** ]
+    tags: [ v* ]
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Install ct
+        uses: helm/chart-testing-action@v2.3.1
+
+      - name: Run lint
+        run: ct lint --config .github/ct.yaml --charts=helm-chart
+
+  install:
+    runs-on: ubuntu-latest
+    services:
+      registry:
+        image: registry:2
+        ports:
+          - 5000:5000
```

Comment:
Right now you have two registries and the config is using a mixture of both. Not sure how your chart install is passing. You're pushing to 5001 but pulling from 5000. You should remove this registry in favor of the embedded one in k3d.

Reply:
Fixed